lib/Agrammon/OutputFormatter/JSON.pm6

```diff
@@ -20,13 +28,13 @@ sub output-for-gui(Agrammon::Model $model,
 #    return ();
 #}
 
-sub get-data($model, $outputs, $include-filters) {
+sub get-data($model, $outputs, $include-filters, $language?, $prints?) {
     my @records;
     for sorted-kv($outputs.get-outputs-hash) -> $module, $_ {
         when Hash {
             for sorted-kv($_) -> $output, $raw-value {
                 my $var = $module ~ '::' ~ $output;
-                push @records, make-record($module, $output, $model, $raw-value, $var);
+                push @records, make-record($module, $output, $model, $raw-value, $var, Any, $language, $prints);
```

Comment:
Might have been better to make the optional things named parameters rather than having to pass the `Any`.

Reply:
Done.